src/CalcManager/Ratpack/ratpak.h

```diff
@@ -80,8 +77,8 @@ typedef struct _number
 
 typedef struct _rat
 {
-    PNUMBER pp;
-    PNUMBER pq;
+    NUMBER pp;
```

Comment:
```suggestion
    NUMBER p;
    NUMBER q;
```

The names `pp` and `pq` are shorthand for `pointer to p` and `pointer to q`.  There will be a lot of change in the codebase if you rename these, but we're also adding confusion now that the names have embedded type information that isn't actually correct.